### DISCUSSION TITLE
Jakarta EE standardization of `io.quarkus.security.PermissionsAllowed`

### DISCUSSION THREAD

**Discussion by jycr** [quarkusio/quarkus#38122] [Community]
For your information, I requested the standardization of the `io.quarkus.security.PermissionsAllowed` annotation in the Jakarta specification (as the `jakarta.annotation.security.RolesAllowed` annotation already is).

-> https://github.com/jakartaee/common-annotations-api/issues/126

**Comment by sberyozkin** [comment]
Thanks @jycr, @AdamBien, FYI, this is something you have considered too as far as I recall

**Reply by sberyozkin** [reply]
@michalvavrik is the author of the Quarkus annotation so please CC him

**Reply by michalvavrik** [reply]
Interesting, let's wait for reactions. Thanks
